tests/model_serving/model_server/model_mesh/test_model_mesh_multi_models.py

```diff
@@ -58,3 +62,25 @@ def test_model_mesh_tensorflow_with_openvino(
             protocol=Protocols.HTTP,
             use_default_query=True,
         )
+
+    def test_product_version_in_model_mesh_container(
+        self,
+        admin_client,
+        http_s3_ovms_model_mesh_serving_runtime,
+        http_s3_openvino_model_mesh_inference_service,
+        http_s3_openvino_second_model_mesh_inference_service,
+    ):
+        """Verify model mesh container contains the correct product version"""
+        pod = get_pods_by_isvc_label(
+            client=http_s3_openvino_model_mesh_inference_service.client,
+            isvc=http_s3_openvino_model_mesh_inference_service,
+            runtime_name=http_s3_ovms_model_mesh_serving_runtime.name,
+        )[0]
+
+        mm_log = pod.log(container="mm")
+
+        if version_match := re.search(r"Registering ModelMesh Service version as \"v(\d\.\d+\.\d+)", mm_log):
+            assert Version(version_match.group(1)) == get_product_version(admin_client=admin_client)
```

Comment:
Are you sure MM version matches RHOAI/ODH version?

Reply:
yes